### DISCUSSION TITLE
Adding Apache Cloudberry Podling Support to DBeaver Community Edition

### DISCUSSION THREAD

**Discussion by edespino** [dbeaver/dbeaver#36071] [DBeaver Development]
Hi DBeaver community,

I'd like to propose adding support for Apache Cloudberry - which recently entered the Apache Incubator program (https://incubator.apache.org/projects/cloudberry.html) - to DBeaver. I have an initial implementation built with all tests passing, using DBeaver's standard tooling. The implementation is based on the existing Greenplum plugin, though simplified as we don't need to handle multiple version support.

For reference, my initial work can be seen here: https://github.com/edespino/dbeaver/tree/apache_cloudberry_incubating
The changes need work to clean up the Greenplum version code, but I wanted to start the discussion with the community about this integration.

FYI: This effort is also being discussed with the Apache Cloudberry community here: https://lists.apache.org/thread/p4zv54jkm54m7qyzp196sxw49dmym0lj

Regards,
-=e

**Comment by edespino** [comment]
As we chart our course forward on the Apache Cloudberry (incubator) project, reaching out to high end user valued open source projects is high on our list. This is my first time interacting with the DBeaver community. Any feedback/guidance is greatly appreciated.

**Comment by edespino** [comment]
I could use some direction from the kind folks at DBeaver on the possibility of supporting Apache Cloudberry (Incubating). We can easily take up the work to provide support in the Community version.

**Reply by Destrolaric** [reply]
@edespino Hi, sorry for waiting. The discussion was lost a bit, so I opened you an issue on our board for better tracking.

**Comment by Destrolaric** [comment]
Hi, I would recommend opening the issue on the DBeaver board so we can track the idea. 
@edespino

**Reply by Destrolaric** [reply]
Created an associated issue https://github.com/dbeaver/dbeaver/issues/36512 I think it is better to continue there

**Comment by tuhaihe** [comment]
Hi all, I saw that the latest DBeaver version already supports Cloudberry (its old name was CloudberryDB. Since Cloudberry joined the Apache incubator, it has been renamed to Apache Cloudberry).

![image](https://github.com/user-attachments/assets/33b3d6a4-c39d-4a47-8877-40b977f57837)

I'm not sure if this thread has any follow-ups.
